IFC4x3/Properties/p/PermeabilityVert_1aKYiWegb7iPXpy4cjfbOY/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="PermeabilityVert_1aKYiWegb7iPXpy4cjfbOY" Name="PermeabilityVert" UniqueId="64522b20-a2a9-47b1-9873-f049ada65622" PrimaryDataType="IfcLinearVelocityMeasure" />
```

Comment:
Propose name "PermeabilityVertical" and data type of course.

Reply:
Same comment as for horizontal.

Reply:
See response for horizontal.